cmd/archive.go

```diff
@@ -115,16 +113,16 @@ An archive is a fully self-contained test run, and can be executed identically e
 	}
 
 	archiveCmd.Flags().SortFlags = false
-	archiveCmd.Flags().AddFlagSet(archiveCmdFlagSet(globalFlags))
+	archiveCmd.Flags().AddFlagSet(archiveCmdFlagSet(&archiveOut))
 
 	return archiveCmd
 }
 
-func archiveCmdFlagSet(globalFlags *commandFlags) *pflag.FlagSet {
+func archiveCmdFlagSet(archiveOut *string) *pflag.FlagSet {
```

Comment:
It seems like we don't need a pointer for this string, don't we? :thinking:

Reply:
we need it here, `flags.StringVarP()` expects a pointer, but I got completely rid of the parameter in https://github.com/grafana/k6/pull/2418 anyway :sweat_smile: